region/client.go

```diff
@@ -265,6 +266,28 @@ func (c *client) inFlightDown() error {
 	return nil
 }
 
+// InFlight returns inFlight value
```

Comment:
Move extra functions to MarshalJSON since they're only needed by it

Reply:
Thanks for moving these functionality, but the original functions still appear on the review